docs/how-to/reactive.md

```diff
@@ -0,0 +1,112 @@
+# How to set up reactive spawning
+
+The charm provides an experimental feature to spawn runners reactively, depending on the jobs requested by GitHub.
+This feature is disabled by default and can be enabled by integrating the charm with a MongoDB database.
+
+## Requirements
+
+You need to deploy a webhook router, which listens for incoming jobs from GitHub, transforms them into
```

Comment:
For my understanding, does this how-to guide have the user deploy the webhook router?

Reply:
yes in line 91

```
juju deploy github-runner-webhook-router --channel latest/edge --revision 30 --config flavours=@routing_table.yaml --config default-flavour=small --config webhook-secret=<your-secret>
```